.github/CONTRIBUTING.md

```diff
@@ -1,5 +1,4 @@
 # Contributing to unfurl bot
-:+1: First off, thanks for taking the time to contribute! :+1:
```

Comment:
Please re-introduce. Positively encouraging community members to contribute to the project, despite the additional effort imposed on them by the bureaucratic steps outlined here, is an important focus.